doc/JSON_INFO.md

```diff
@@ -3,28 +3,115 @@
 ## File descriptions
 Here's a quick summary of what each of the JSON files contain, broken down by folder.
 
-## `raw/`
-
-| Filename               | Description
-|---                     |---
-| bionics.json           | bionics, does NOT include bionic effects
-| dreams.json            | dream text and linked mutation categories
-| item_groups.json       | item spawn groups
-| materials.json         | material types
-| monstergroups.json     | monster spawn groups
-| monster_factions.json  | monster factions
-| names.json             | names used for NPC/player name generation
-| professions.json       | profession definitions
-| recipes.json           | crafting/disassembly recipes
-| road_vehicles.json     | vehicle spawn information for roads
-| skills.json            | skill descriptions and ID's
-| snippets.json          | flier/poster descriptions
-| mutations.json         | traits/mutations
-| mutation_ordering.json | draw order for mutation overlays in tiles mode
-| vehicle_groups.json    | vehicle spawn groups
-| vehicle_parts.json     | vehicle parts, does NOT affect flag effects
-
-## `raw/vehicles/`
+## `data/json/`
+
+| Filename                    | Description
+|---                          |---
+| anatomy.json                | a listing of player body parts - do not edit
+| bionics.json                | bionics, does NOT include bionic effects
+| body_parts.json             | an expansion of anatomy.json - do not edit
+| construction.json           | definition of construction menu tasks
+| default_blacklist.json      | a standard blacklist of joke monsters
+| doll_speech.json            | talk doll speech messages
+| dreams.json                 | dream text and linked mutation categories
+| effects.json                | common effects and their effects
+| emit.json                   | smoke and gas emissions
+| flags.json                  | common flags and their descriptions
+| furniture.json              | furniture, and features treated like furniture
+| game_balance.json           | various options to tweak game balance
+| gates.json                  | gate terrain definitions
+| harvest.json                | item drops for butchering corpses
+| health_msgs.json            | messages displayed when the player wakes
+| item_actions.json           | descriptions of standard item actions
+| item_category.json          | item categories and their default sort
+| item_groups.json            | item spawn groups
+| lab_notes.json              | lab computer messages
+| martialarts.json            | martial arts styles and buffs
+| materials.json              | material types
+| monster_attacks.json        | monster attacks
+| monster_drops.json          | monster item drops on death
+| monster_factions.json       | monster factions
+| monstergroups.json          | monster spawn groups
+| monstergroups_egg.json      | monster spawn groups from eggs
+| monsters.json               | zombie descriptions
```

Comment:
Not only zombie, there are a lot of other monster types too.

Reply:
in monsters.json it's almost entirely zombies.
other monsters are in monsters/*json